homeassistant/components/vicare/climate.py

```diff
@@ -84,17 +75,17 @@
 }
 
 VICARE_TO_HA_PRESET_HEATING = {
-    VICARE_PROGRAM_COMFORT: PRESET_COMFORT,
-    VICARE_PROGRAM_ECO: PRESET_ECO,
-    VICARE_PROGRAM_NORMAL: PRESET_HOME,
-    VICARE_PROGRAM_REDUCED: PRESET_SLEEP,
+    Program.COMFORT: PRESET_COMFORT,
+    Program.ECO: PRESET_ECO,
+    Program.NORMAL: PRESET_HOME,
+    Program.REDUCED: PRESET_SLEEP,
 }
 
 HA_TO_VICARE_PRESET_HEATING = {
```

Comment:
`HA_TO_VICARE_PRESET_HEATING = {v: k for k, v in VICARE_TO_HA_PRESET_HEATING.items()}`